install/kubernetes/helm/istio/charts/pilot/templates/gateway.yaml

```diff
@@ -1,3 +1,58 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istio-autogenerated-k8s-ingress
+  namespace: istio-system
+spec:
+  selector:
+    istio: {{ .Values.global.ingressSelector }}
+  servers:
+  - port:
+      number: 80
+      protocol: HTTP
+      name: http
+    hosts:
+    - "*"
+  - port:
+      number: 443
+      protocol: HTTPS
+      name: https-default
+    tls:
+      mode: SIMPLE
+      serverCertificate: /etc/istio/ingress-certs/tls.crt
+      privateKey: /etc/istio/ingress-certs/tls.key
+    hosts:
+    - "*"
+
+---
+
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: ingressgateway
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
```

Comment:
why add this? This will cause trouble with the hosts in virtual services

Reply:
Why ? If a VirtualService is pointing to ingressgateway - will get added. Otherwise - it will use whatever Gateway is defined. 

What troubles do you expect ?

Reply:
We already have a gateway, which one is this?

Reply:
If this is a Gateway to be used for services like grafanat, prometheus etc, I suggest we name it "istio-services-gateway".